gateway/app/settings.py

```diff
@@ -35,13 +35,21 @@ class SMSOTPSettings(BaseModel):
     sender_id: Annotated[str, Field(max_length=11)]
 
 
+class ProvisioningSettings(BaseModel):
+    nef_url: AnyHttpUrl
+    gateway_url_callback: AnyHttpUrl
+    gateway_url: AnyHttpUrl
```

Comment:
This should be moved to the main settings as it will be used in more places than here